modules/api/src/main/java/org/apache/fluo/api/config/FluoConfiguration.java

```diff
@@ -716,26 +784,51 @@ public FluoConfiguration clearObservers() {
     return this;
   }
 
+  /**
+   * Sets the transaction rollback time, in milliseconds. 
+   * @param time A long representation of the duration
+   * @param tu The TimeUnit to use
+   */
   public FluoConfiguration setTransactionRollbackTime(long time, TimeUnit tu) {
     return setPositiveLong(TRANSACTION_ROLLBACK_TIME_PROP, tu.toMillis(time));
   }
-
+  /**
+   * Gets the transaction rollback time, in milliseconds.
```

Comment:
need to mention TRANSACTION_ROLLBACK_TIME_PROP and TRANSACTION_ROLLBACK_TIME_DEFAULT